### ISSUE TITLE
Gradle plugin deprecation warnings in Gradle 6.4

### ISSUE THREAD

**Issue by niktekusho** [dependency-check/DependencyCheck#2644]
**Describe the bug**
Not really a bug. Just a heads up for upcoming Gradle releases. :wink:
When running the dependency analyzer plugin in a multi-project setup, Gradle warns about the resolution of deprecated configurations. Gradle 7 will not set up these configurations.

Example:
```
The archives configuration has been deprecated for resolution. This will fail with an error in Gradle 7.0. Please resolve the compileClasspath or runtimeClasspath configuration instead. Consult the upgrading guide for further information: https://docs.gradle.org/6.4/userguide/upgrading_version_5.html#dependencies_should_no_longer_be_declared_using_the_compile_and_runtime_configurations
```

You can find a reproduction repository here: https://github.com/niktekusho/owasp-gradle-plugin-deprecation-sample.

**Version of dependency-check used**
Gradle Plugin `5.3.2` with Gradle `6.4`

**Log file**
```
> Task :dependencyCheckAnalyze
Verifying dependencies for project owasp-gradle-plugin-deprecation-sample
The archives configuration has been deprecated for resolution. This will fail with an error in Gradle 7.0. Please resolve the compileClasspath or runtimeClasspath configuration instead. Consult the upgrading guide for further information: https://docs.gradle.org/6.4/userguide/upgrading_version_5.html#dependencies_should_no_longer_be_declared_using_the_compile_and_runtime_configurations
The compile configuration has been deprecated for resolution. This will fail with an error in Gradle 7.0. Please resolve the compileClasspath configuration instead. Consult the upgrading guide for further information: https://docs.gradle.org/6.4/userguide/upgrading_version_5.html#dependencies_should_no_longer_be_declared_using_the_compile_and_runtime_configurations
The compileOnly configuration has been deprecated for resolution. This will fail with an error in Gradle 7.0. Please resolve the compileClasspath configuration instead. Consult the upgrading guide for further information: https://docs.gradle.org/6.4/userguide/upgrading_version_5.html#dependencies_should_no_longer_be_declared_using_the_compile_and_runtime_configurations
The default configuration has been deprecated for resolution. This will fail with an error in Gradle 7.0. Please resolve the compileClasspath or runtimeClasspath configuration instead. Consult the upgrading guide for further information: https://docs.gradle.org/6.4/userguide/upgrading_version_5.html#dependencies_should_no_longer_be_declared_using_the_compile_and_runtime_configurations
The runtime configuration has been deprecated for resolution. This will fail with an error in Gradle 7.0. Please resolve the runtimeClasspath configuration instead. Consult the upgrading guide for further information: https://docs.gradle.org/6.4/userguide/upgrading_version_5.html#dependencies_should_no_longer_be_declared_using_the_compile_and_runtime_configurations
Checking for updates and analyzing dependencies for vulnerabilities
Generating report for project owasp-gradle-plugin-deprecation-sample
Found 0 vulnerabilities in project owasp-gradle-plugin-deprecation-sample
```

**To Reproduce**
You can find a reproduction repository here: https://github.com/niktekusho/owasp-gradle-plugin-deprecation-sample.

**Expected behavior**
No deprecation warnings are shown by Gradle.

**Comment by lawrenceong** [comment]
Tried it with gradle `6.6.1` and plugin version `6.0.2` and the issue still exists.

**Comment by jeremylong** [comment]
The fix will be included in 6.0.3.

**Comment by jeremylong** [comment]
To be clear - the warning will appear if using gradle 6.4. However, if using gradle 7.0 or higher ODC will no longer try to resolve these scopes.
